blockstream/test/adapter.test.ts

```diff
@@ -0,0 +1,88 @@
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+  process.env.API_KEY = process.env.API_KEY ?? 'test_api_key'
+
+  context('successful calls @integration', () => {
+    const requests = [
+      {
+        name: 'empty data',
+        testData: { data: {} },
+      },
+      {
+        name: 'id not supplied',
+        testData: {
+          data: {
+            endpoint: 'ethgasAPI',
```

Comment:
These docs also look to be carried over from EthGasStation and need to be updated for Blockstream.